src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -23,10 +23,17 @@ const ClaimsRow: React.FC<ClaimsRowData> = ({ claim, group, children }) => {
   return (
     <tr>
       <td>
-        <Text>{claim.distribution.epoch.circle?.organization?.name}</Text>
+        <Flex>
+          {claim.distribution.epoch.circle?.logo ? (
+            <Avatar small path={claim.distribution.epoch.circle?.logo} />
```

Comment:
this has a margin around it - and it's aligning wrong - so I need to remove all the margin except the right margin... not sure cleanest way to do this with stitches
<img width="1352" alt="Screen Shot 2022-08-05 at 4 22 25 PM" src="https://user-images.githubusercontent.com/83605543/183222950-551c5fb0-1f65-469c-a9fc-e7725400e5c6.png">

Reply:
Avatar is written to pass all props not otherwise specified to the AvatarRoot component, which takes the standard `css` prop, so adding something like `css={{ my: 0, ml: 0 }}` should do the trick

Reply:
fixed this with help from @CryptoGraffe - but its kinda ugly.... and have to pass cssProps manually through. @levity

Reply:
oh that's interesting. you could inline your ternary if you want it to be a bit more compact:
```
{...(css ? { css } : {})}
```